docs/sources/flow/reference/components/loki.source.file.md

```diff
@@ -0,0 +1,109 @@
+---
+aliases:
+- /docs/agent/latest/flow/reference/components/loki.source.file
+title: loki.source.file
+---
+
+# loki.source.file
+
+`loki.source.file` reads log entries from files and forwards them to other
+`loki.*` components.
+
+Multiple `loki.source.file` components can be specified by giving them
+different labels.
+
+## Usage
+
+```river
+loki.source.file "LABEL" {
+  targets    = TARGET_LIST
+  forward_to = RECEIVER_LIST
+}
+```
+
+## Arguments
+The component starts a new reader for each of the given `targets` and fans out
+log entries to the list of receivers passed in `forward_to`.
+The list of arguments that can be used to configure the block is presented
+below.
```

Comment:
```suggestion

`loki.source.file` supports the following arguments:
```

Reply:
Suggestion for simplifying and making this consistent with other component Arguments sections.